askama_shared/src/lib.rs

```diff
@@ -293,33 +292,6 @@ static DEFAULT_ESCAPERS: &[(&[&str], &str)] = &[
     (&["j2", "jinja", "jinja2"], "::askama::Html"),
 ];
 
-#[derive(Debug)]
```

Comment:
I don't like this. I think we should have a custom error type. We could redefine it as `CompileError(Cow<'static, str>)` and I agree that that would be an improvement. I would probably do it in two steps: use the `From` impls in favor of literal `Static` and `String` variants, then change the inner type. How about we take that as a separate PR for now to help shrink this one?